styles/main.css

```diff
@@ -186,9 +186,6 @@ a:hover .playButton {
   opacity: 1;
 }
 
-.navbar{
-  height:45px!important;
```

Comment:
bootstrap has a min-height of 60. which overrules height. You could alternatively also set min-height here but I like the 60 tall navbar.